packages/core/src/auth/secondaryAuth.ts

```diff
@@ -156,6 +157,12 @@ export class SecondaryAuth<T extends Connection = Connection> {
     }
 
     public get isConnectionExpired() {
+        if (this.activeConnection) {
+            getLogger().info(
+                indent(`secondaryAuth connection id = ${this.activeConnection.id}
+            secondaryAuth connection status = ${this.auth.getConnectionState(this.activeConnection)}`)
```

Comment:
we probably want to use the `clear` parameter of `indent()`.
```suggestion
            secondaryAuth connection status = ${this.auth.getConnectionState(this.activeConnection)}`,
            4, true)
```